config.cson

```diff
@@ -2850,6 +2850,11 @@ fileIcons:
 		match: ".pogo"
 		colour: "auto-orange"
 
+	"Polymer.json":
+		icon: "polymer"
+		match: "polymer.json"
+		color: "purple"
```

Comment:
Use this instead:

~~~coffeescript
	Polymer:
		icon: "polymer"
		priority: 2
		match: /^polymer\.json$/i
		colour: "medium-purple"
~~~

- `priority`: If this isn't included, the pattern for normal ".json" files will match first.
- `match`: `"polymer.json"` also matches `some-other-polymer.json`, because the config treats strings as shorthand forms of `/string$/i`.
- `colour`: A shade needs to be specified, like `dark-`, `medium-`, etc.